resources/log4j.properties

```diff
@@ -17,6 +17,7 @@ log4j.appender.file.layout.ConversionPattern=%d [%t] %-5p%c - %m%n
 log4j.appender.metabase=metabase.logger.Appender
 
 # customizations to logging by package
+log4j.logger.com.mchange.v2.resourcepool=ERROR # c3p0 connection pools tend to log useless warnings way too often; only log actual errors
 log4j.logger.metabase=INFO
 log4j.logger.metabase.db=DEBUG
```

Comment:
I have removed this line on several occasions but it's clear you feel this is a required part of the normal running of the app.  Can you explain your reasoning for that?  I've never seen an application that purposely ships with a logs setting which prints out EVERY database call the app ever makes.

Same goes for the rest of these DEBUG log entries really.  I think a case could be made for some of the driver/qp code while the app is still young and developing, but normally an app wouldn't ship with any DEBUG logging enabled.

Either way, it would make sense to better understand what the logging strategy for the app is intended to be @tlrobinson @salsakran